docs/bcfusion/users/new-stake.md

```diff
@@ -0,0 +1,78 @@
+import Stake1 from '../../assets/bcfusion/user-stake1.png';
+import Stake2 from '../../assets/bcfusion/user-stake2.png';
+import Stake3 from '../../assets/bcfusion/user-stake3.png';
+import Stake4 from '../../assets/bcfusion/user-stake4.png';
+import Stake5 from '../../assets/bcfusion/user-stake5.png';
+import Stake6 from '../../assets/bcfusion/user-stake6.png';
+import Stake7 from '../../assets/bcfusion/user-stake7.png';
+import Stake8 from '../../assets/bcfusion/user-stake8.png';
+import Stake9 from '../../assets/bcfusion/user-stake9.png';
+import Stake10 from '../../assets/bcfusion/user-stake10.png';
+
+# New Stake Management
+
+You can new the BNB staking dApp to manage your staks easily.
+
+BNB Staking dApp on testnet: https://testnet-staking.bnbchain.org/en/bnb-staking
+
+BNB Staking dapp on mainnet: To be updated.
+
+## Connect Wallet
+
+<img src={Stake1} width="400"/>
+
+Firstly, you need to connect to a web3 wallets for mananging your stakes.
+`Trust` (mainnet only now) and `MetaMask` wallets are support, as well as other wallets which support `WalletConnect`.
+
+## Delegate
+
+Then you can choose a validator you would like to delegate to. You can get more information by viewing the validator
+page
+for each validator.
+
+Click the `Delegate` button to start creating a new delegation.
+
+<img src={Stake2} width="400"/>
+
+Input the BNB amount you would like to delegate.
+
+<img src={Stake3} width="400"/>
+
+After clicking `Delegate` button, the wallet you connect will ask you for signing the transaction.
+
+And if the transaction goes well, you will find your new delegation in `My Staking` page with transaction hash.
+
+<img src={Stake4} width="400"/>
+
+# Redelegate
+
+In `My Staking` page you can manage you existing delegations.
+
+By clicking `Redelegate` button, you can redelegate your stake to another validator.
+
+<img src={Stake6} width="400"/>
+
+In the popup window, you can choose the destination validator your want to stake to, as well as the amount you want to
```

Comment:
please hightlight that the redelegate will charge 0.02% fee.

Reply:
Sure, added